src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -1198,6 +1250,43 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+input PlanDebugSettings {
+    itineraryFilterDebugProfile: ItineraryFilterDebugProfile = OFF
+}
+
+input PlanLocation {
+    coordinate: Coordinate
+    stop: FeedScopedId
+    label: String
+}
+
+input PlanPreferences {
+    street: PlanStreetPreferences
+    transit: TransitPreferences
+    accessibility: AccessibilityPreferences
+}
+
+input PlanStreetModes {
+    """Selection of only one allowed for now."""
+    direct: [PlanDirectMode!]
+    """Selection of only one allowed for now."""
+    access: [PlanAccessEgressMode!]
+    """Selection of only one allowed for now."""
+    egress: [PlanAccessEgressMode!]
+    """Selection of only one allowed for now."""
+    transfer: [PlanTransferMode!]
+}
+
+input PlanStreetPreferences {
+    modes: PlanStreetModes
+    bike: BikePreferences
+    scooter: ScooterPreferences
+    car: CarPreferences
+    walk: WalkPreferences
+    rental: RentalPreferences
+    parking: VehicleParkingInput
```

Comment:
Should we reuse this `VehicleParkingInput` or create a new type? The `VehicleParkingInput` had some known limitations but on the other hand, I'm not sure if anyone has interest any time soon to make it more generic.